### PR TITLE
Updates to migrate to new subscription. 

### PR DESCRIPTION
**Pull request by marrobi**
# PR for issue #1125

## What is being addressed

Work to migrate to new subscription.

### CODE REVIEW

**Comment by marrobi** [comment]
Providing this goes green, https://github.com/microsoft/AzureTRE/actions/runs/1720690262 , this is good to go.

File: templates/core/terraform/scripts/letsencrypt.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+echo $(env)
```
Comment (martinpeck) [inline templates/core/terraform/scripts/letsencrypt.sh:4]: remove this please

Comment (martinpeck) [reply templates/core/terraform/scripts/letsencrypt.sh:4]: sorry....what I meant to say is...

can you explain what this does? 

(soz)

**Review by martinpeck — CHANGES_REQUESTED** [review]

**Review by martinpeck — APPROVED** [review]
LGTM

**Pull request merged**